modules/react/modal/stories/examples/Basic.tsx

```diff
@@ -1,31 +1,36 @@
 import React from 'react';
 
 import {Modal} from '@workday/canvas-kit-react/modal';
-import {DeleteButton} from '@workday/canvas-kit-react/button';
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
 import {HStack, Box} from '@workday/canvas-kit-react/layout';
 
 export const Basic = () => {
-  const handleDelete = () => {
-    console.log('Deleted item');
+  const handlePrimaryClick = () => {
+    console.log('Primary Action clicked');
+  };
+
+  const handleSecondaryClick = () => {
+    console.log('Secondary Action clicked');
   };
 
   return (
     <Modal>
-      <Modal.Target as={DeleteButton}>Delete Item</Modal.Target>
+      <Modal.Target as={PrimaryButton}>Open License</Modal.Target>
       <Modal.Overlay>
         <Modal.Card>
           <Modal.CloseIcon aria-label="Close" />
-          <Modal.Heading>Delete Item</Modal.Heading>
+          <Modal.Heading>MIT License</Modal.Heading>
           <Modal.Body>
             <Box as="p" marginY="zero">
-              Are you sure you want to delete the item?
+              Permission is hereby granted, free of charge, to any person obtaining a copy of this
+              software and associated documentation files (the "Software").
             </Box>
           </Modal.Body>
           <HStack spacing="s" padding="xxs" marginTop="xxs">
-            <Modal.CloseButton as={DeleteButton} onClick={handleDelete}>
-              Delete
+            <Modal.CloseButton as={PrimaryButton} onClick={handlePrimaryClick}>
+              Primary Action
             </Modal.CloseButton>
-            <Modal.CloseButton>Cancel</Modal.CloseButton>
+            <Modal.CloseButton onClick={handleSecondaryClick}>Secondary Action</Modal.CloseButton>
```

Comment:
I feel like it would be nice if the primary and secondary buttons corresponded to real-world actions rather than the generic `Primary Action` and `Secondary Action`, much like how you have `Acknowledge` and `Cancel` in the [Custom Focus](https://github.com/Workday/canvas-kit/pull/1621/files#diff-ab1bb73b5ce0a94b70381fae7f8cd55ec96167dbcf780c33a5abfd1b784755c3R35) example. Thoughts?

Reply:
Makes sense to me! I'll change it!